### PR TITLE
feat: DAH-2621 add unleash to rails

### PR DESCRIPTION
**Pull request by tallulahkay**
## Description

Adds unleash flags to rails server. Please also review https://github.com/SFDigitalServices/sf-dahlia-credentials/pull/33 at the same time.

## Jira ticket

DAH-2621

### Version Control

- [ ] branch name begins with `angular` if it contains updates to Angular code
- [x] branch name contains the Jira ticket number
- [x] PR name follows `type: TICKET-NUMBER Description` format, e.g. `feat: DAH-123 New Feature`

### Code quality

- [x] [the set of changes is small](https://google.github.io/eng-practices/review/developer/small-cls.html#what-is-small)
- [ ] [if the set of changes cannot be small, reviewers have been forewarned](https://google.github.io/eng-practices/review/developer/small-cls.html#cant)
- [x] code meets test coverage thresholds
- [x] code is properly formatted
- [x] code is linted
- [x] code irrelevant to the ticket is not modified e.g. changing indentation due to automated formatting
- [x] automated tests pass
- [x] if the code changes the UI, it matches the UI design exactly

### Review instructions

- [x] instructions specify which environment(s) it applies to
- [x] instructions have already been performed at least once
- [x] instructions can be followed by PA testers
- [x] instructions specify if it can only be followed by an engineer

### Request review

- [x] PR has `needs review` label
- [x] Use `Housing Eng` group to automatically assign reviewers, and/or assign specific engineers
- [ ] If time sensitive, notify engineers in Slack

### CODE REVIEW

**Review by chadbrokaw — APPROVED** [review]

File: lib/tasks/translate_test.rake [hunk 1]
```diff
@@ -57,4 +57,12 @@ namespace :translate_test do # rubocop:disable Metrics/BlockLength
   task unsubscribe: :environment do
     Rails.cache.write(Force::EventSubscriberTranslateService::UNSUBSCRIBE_CACHE_KEY, true)
   end
+
+  task flag: :environment do
```
Comment (jimlin-sfgov) [inline lib/tasks/translate_test.rake:61]: suggestion: Also add a `TODO` comment here.

**Review by jimlin-sfgov — APPROVED** [review]
one suggested change, but otherwise LGTM

**Comment by jimlin-sfgov** [comment]
suggestion: This came to me when I saw the build errors: provide [default values](https://ruby-doc.org/3.2.2/ENV.html#method-c-fetch) when fetching env vars to prevent build errors, e.g. `ENV.fetch('UNLEASH_URL', nil)`

File: app/services/force/listing_service.rb [hunk 2]
```diff
@@ -33,6 +33,13 @@ def self.listing(id, opts = {})
       Rails.logger.info("Calling self.listing for #{id} with force: #{force}")
       results = Request.new(parse_response: true).cached_get(endpoint, nil, force)
       results_with_cached_listing_images = add_cloudfront_urls_for_listing_images(results)
+      # TODO: DAH-2636 conditionally get translations using feature flag
+      if ::UNLEASH.is_enabled? 'test'
+        puts 'test is enabled'
```
Comment (jimlin-sfgov) [inline app/services/force/listing_service.rb:38]: thought: We can ignore the coverage issue and bypass it to merge this PR, but in the follow-up ticket we should update the specs for this method.

File: lib/tasks/translate_test.rake (hunk 1 above)
Comment (tallulahkay) [reply lib/tasks/translate_test.rake:61]: I actually didn't mean to leave this in here - it doesn't work

**Pull request merged**